### PR TITLE
Fork at 257000 per Unforkening Nutocracy motion,

### PR DESCRIPTION
**Pull request by FreeTrade**
https://nutocracy.herokuapp.com/arguments/that-it-is-time-to-un-fork-hodlcoin-and-stop-auto-hodling-for-a-year

Autohodling removed from block 257000

Older version clients disconnected after block 250000

Also, bug fix to ensure interest not earned on deposits after maturation period ends.

### CODE REVIEW

File: src/primitives/transaction.h [hunk 1]
```diff
@@ -11,6 +11,8 @@
 #include "serialize.h"
 #include "uint256.h"
 
+static const unsigned int THEUNFORKENING = 257000;
```
Comment (Fuzzbawls) [inline src/primitives/transaction.h]: seems like an odd place for this kind of constant as it not really a transaction primitive but a consensus rule

File: src/version.h [hunk 2]
```diff
@@ -9,7 +9,7 @@
  * network protocol versioning
  */
 
-static const int PROTOCOL_VERSION = 70010;
+static const int PROTOCOL_VERSION = 250000;
```
Comment (Fuzzbawls) [inline src/version.h]: is there a reason for such a massive jump in protocol version?

File: src/version.h [hunk 3]
```diff
@@ -22,7 +22,7 @@ static const int MIN_PEER_PROTO_VERSION = GETHEADERS_VERSION;
 
 //! disconnect from peers once fork height reached
 static const int MIN_PEER_PROTO_MINERHODLFORK_VERSION = 70010;
-
+static const int MIN_PEER_PROTO_THEUNFORKENING_VERSION = 250000;
```
Comment (Fuzzbawls) [inline src/version.h]: See comment above

File: src/primitives/transaction.h (hunk 1 above)
Comment (FreeTrade) [inline src/primitives/transaction.h]: Yeah, I initially had it in main.h, but found it was required in the interest calculation code here to fix the interest rate bug.

File: src/version.h (hunk 2 above)
Comment (FreeTrade) [inline src/version.h]: I had previously thought that the version here was doubling as the block at which clients using the old code were disconnected.

**Pull request merged**